docs/designs/template.md

```diff
@@ -0,0 +1,27 @@
+# Proposal Title
+
+* Proposal: < GITHUB ISSUE URL >
+* See Also:
+  * < RELEVANT URL >
+  * < RELEVANT URL >
+
+## Abstract
+
+< Provide a short summary of what would be achieved and why is is of value >
+
+< If user stories are relevant, add them here >
+
+## Proposal
+
+< Provide a detailed proposal of **what** would be done and **how**; feel free to add multiple
+subsections, diagrams, and config/code snippets >
+
```

Comment:
Might be nice to have a section that explicitly calls out use cases or scenarios that justify why this change is needed. I've seen that work well to frame the discussion and help illustrate why certain choices are being made.

Reply:
Sub-level of abstract around user-stories and why the change should be made.

Reply:
Add a `Why` subsection to the abstract.

If this satisfies the request, please resolve this comment. Thanks!

Reply:
@stmcginnis @joshrosso i'll taking the liberty of adding more things in the template.